start.py

```diff
@@ -956,10 +957,17 @@ def sigterm_handler(_signo, _stack_frame):
 
     signal.signal(signal.SIGTERM, sigterm_handler)
 
-    service_backups()
-    set_up_nginx_files(m2ee)
-    complete_start_procedure_safe_to_use_for_restart(m2ee)
-    set_up_instadeploy_if_deploy_password_is_set(m2ee)
-    start_metrics(m2ee)
-    start_nginx()
-    loop_until_process_dies(m2ee)
+    try:
+        service_backups()
+        set_up_nginx_files(m2ee)
+        complete_start_procedure_safe_to_use_for_restart(m2ee)
+        set_up_instadeploy_if_deploy_password_is_set(m2ee)
+        start_metrics(m2ee)
+        start_nginx()
+        loop_until_process_dies(m2ee)
+    except Exception:
+        x = traceback.format_exc()
+        logger.error('Starting app container failed: %s' % x)
+        callback_url = os.environ.get('BUILD_STATUS_CALLBACK_URL')
+        if callback_url:
+            requests.put(callback_url, x)
```

Comment:
`raise` is done.

On the other one, so maybe I am missing something but we really do not know or cannot guarantee
that nothing will break. This is my intention that if anything breaks, record at S3 via Deployer such
that it should be still able to pick up `PackageState.FAILED` from CF and read from S3 about why.

If we wrap the exception handling around `complete_start_procedure_safe_to_use_for_restart`,
I am not sure we cover other failures.